price-feeder/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/umee-network/umee/price-feeder
+module github.com/umee-network/umee/v2/price-feeder
```

Comment:
The v2 is for the umee part, the price-feeder is in its own version like 0.20 

Maybe it wouldn't be better for the price-feeder to have its own repository? 
@toteki @adamewozniak

Reply:
Long term, separate repo could simplify things.

As for the `price-feeder/go.mod` import path, this PR might work without (since only the umee go.mod, not the whole umee repository, has changed import paths).